code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -741,14 +747,13 @@
 	results = list(/datum/reagent/mutationtoxin/oozeling = 5)
 	required_reagents  = list(/datum/reagent/mutationtoxin/unstable = 5, /datum/reagent/medicine/calomel = 10, /datum/reagent/toxin/bad_food = 30, /datum/reagent/stable_plasma = 5)
 
-
-//////////////Mutatuion toxins made out of advanced toxin/////////////
-
 /datum/chemical_reaction/mutationtoxin/skeleton
 	name = /datum/reagent/mutationtoxin/skeleton
 	id = /datum/reagent/mutationtoxin/skeleton
 	results = list(/datum/reagent/mutationtoxin/skeleton = 5)
-	required_reagents  = list(/datum/reagent/aslimetoxin = 5, /datum/reagent/consumable/milk = 30, /datum/reagent/toxin/acid/fluacid = 30) //Because acid melts flesh off.
+	required_reagents  = list(/datum/reagent/mutationtoxin/unstable = 5, /datum/reagent/consumable/milk = 30, /datum/reagent/toxin/acid/fluacid = 30) //Because acid melts flesh off.
```

Comment:
Non documented change 😔

Reply:
oh whoops.  That's from a different pull request, https://github.com/BeeStation/BeeStation-Hornet/pull/4177.

How would I go about splitting them?

Reply:
Yeah your other pull has the apid stuff as well, you should close one of the two and redo the changes in different branches

Reply:
I'll just nix the skelly stuff for now and leave the Apid stuff in.